raiden/tests/unit/test_channelstate.py

```diff
@@ -1865,19 +1880,18 @@ def test_receive_withdraw_confirmation():
             EventInvalidReceivedWithdraw,
             {
                 "attempted_withdraw": 100,
-                "reason": "Total withdraw confirmation 100 does not match our total withdraw 0",
+                "reason": "Received withdraw confirmation 100 was not found in withdraw states",
```

Comment:
IMO we should not ever assert on any messages, this is very brittle and honestly it doesn't make the test any better.